features/bootstrap/FeatureContext.php

```diff
@@ -856,7 +856,14 @@ public function install_wp( $subdir = '' ) {
 			if ( $install_cache_path ) {
 				mkdir( $install_cache_path );
 				self::dir_diff_copy( $run_dir, self::$cache_dir, $install_cache_path );
-				self::run_sql( 'mysqldump --no-defaults', array( 'result-file' => "{$install_cache_path}.sql" ), true /*add_database*/ );
+
+				$support_column_statistics = exec( '/usr/bin/env mysqldump --help | grep "column-statistics"' );
+				$command                   = '/usr/bin/env mysqldump --no-defaults';
```

Comment:
We cannot actually add `/usr/bin/env` here as it will be automatically added down the road on non-Windows platforms and skipped on Windows.
```suggestion
				$command                   = 'mysqldump --no-defaults';
```